samples/pmic/npmx/charger_and_events/README.rst

```diff
@@ -0,0 +1,97 @@
+.. _npmx_charger_and_events_sample:
+
+npmx: CHARGER AND EVENTS
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The npmx CHARGER AND EVENTS sample demonstrates how to control PMIC CHARGER and receive events using npmx drivers.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf5340dk_nrf5340_cpuapp
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+This sample uses CHARGER and EVENTS driver to do the following:
+
+  * Set CHARGER basic configuration.
+  * Register EVENTS from device.
+
+Wiring
+******
+
+#. Connect the TWI interface between the chosen DK and the nPM1300 EK as in the following table:
+
+.. list-table:: nPM1300 EK connections.
+   :widths: 25 25 25
+   :header-rows: 1
+
+   * - nPM1300 EK
+     - nRF5340 DK
+     - nRF52840 DK
+   * - GPIO0
+     - P1.10
+     - P1.10
+   * - SDA
+     - P1.02
+     - P0.26
+   * - SCL
+     - P1.03
+     - P0.27
+   * - VOUT2 & GND
+     - External supply (P21)
+     - External supply (P21)
```

Comment:
please correct the indentation for lines 36-54 - missing 3 spaces at the beginning of each line